packages/atst/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@eth-optimism/atst",
+  "version": "0.0.0",
```

Comment:
should this be 0.0.1 ?

Reply:
Going to patch it to 0.0.1 with a changeset in the final release pr